Automattic/jetpack
# Update of Out of Sync Site Health message to be more clear.

Currently we output an issue in Site Health when the Jetpack site has experienced a "Data Loss" event and we know that cache data is out of alignment. To resolve this issue currently users needs to trigger a Full Sync which can be done from the action. However the message is a little vague on what exactly is wrong.

![screenshot-2020-11-03-at-15 17 47](https://user-images.githubusercontent.com/5784663/98563508-42adb680-2279-11eb-9264-a0a6e792a926.png)

Therefore we are updating the message to "Jetpack has detected that data is not properly in sync which may be impacting some of your site's functionality."

#### Changes proposed in this Pull Request:
* Update the Site Health test `test__sync_health`

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Checkout or Apply this PR to your test site.
* Go to the Site Health page under Tools
* In CLI execute the following command `\Automattic\Jetpack\Sync\Health::update_status( 'out_of_sync' );`
* Refresh the Site Health page. 
* Expand the "Jetpack has detected an error syncing your site issue.
* Verify the issue text reads "Error Jetpack has detected that data is not properly in sync which may be impacting some of your site's functionality. We recommend a full sync to align Jetpack with your site data." next to the red x.
Click on "Full Sync".
* To clean up after your testing either reset the status to "in_sync" or perform a Full Sync.

#### Proposed changelog entry for your changes:
* Jetpack Sync : Updated Site Health description for better usability.
